### PR TITLE
Preserve key order in galaxy.yml

### PR DESCRIPTION
**Pull request by mkrizek**

### CODE REVIEW

File: migrate.py [hunk 1]
```diff
@@ -282,7 +282,7 @@ def assemble_collections(spec, args):
             'license': None,
             'license_file': None,
             'tags': None,
-            'dependencies': {},
+            'dependencies': None,
             'repository': None,
```
Comment (webknjaz) [inline migrate.py]: Why is this necessary?

Comment (mkrizek) [reply migrate.py]: It isn't. Wanted consistency, but I'll be redoing that code shortly anyway, so going for simple change now. Updated.

**Review by webknjaz — APPROVED** [review]
